ep001/hello.py

```diff
@@ -0,0 +1,169 @@
+
+# original from: PanosTrak
+print('Hello Sexy Boy Anthony')
+
+
+# === Rev. 1 ===
```

Comment:
I would maybe do these in separate files otherwise these aren't runnable

Reply:
I actually tested this. And it runs correctly. But you have to comment up everything else up to the current revision :D
But if you want - I can separate it into different files.

I thought that this way will be less confusing for any the beginners, because it preserves the exact filename from the video.
But maybe not - maybe it hurts readability too much.

Reply:
subdirs then maybe? `rev1/hello.py` `rev1/hello_test.py`

Reply:
I can imagine, you posting a 25-30 min video :D It will have 30+ revisions. But subfolders seems reasonable to me.
I split them for now and if we come up with a better idea - oh well, I will change them again.